core/spec.md

```diff
@@ -5546,6 +5588,42 @@ as defined below:
   - MUST NOT be present if the Resource's `hasdocument` model attribute is
     set to `false.
 
+##### `ancestor` Attribute
+- Type: String
+- Description: The `versionid` of this Version's ancestor.
+
+  The `ancestor` attribute MUST be set to any existing Version's `versionid`,
```

Comment:
...MUST be set to the `versionid` of this Version's ancestor (parent). If this Version is a root of an ancestor hierarchy tree (i.e. it has no parent) then it MUST be set to its own `versionid` value.